Merge WooCommerce Google Docs guide with README.md for WooCommerce github repository (#10)
urbitassociates/WooCommerce-Urbit-Delivery: README.md

The module is installed on your Woocommerce shop.

## Installation

### Getting a plugin

> The plugin is not published in any marketplace yet so the only way to install it right now is to upload a **.zip** file from the Urb-it’s github repository. 

Reviewer: ```suggestion
The plugin is not published in any marketplace yet. Thus, the only way to install it is to upload a **.zip** file from the Urb-it’s github repository. 
```

1. Click **Clone or download** (green button on the right side).
2. Push a **Download ZIP** button and the archive will be downloaded to your local machine.
3. Go to your Wordpress store back office page (admin panel).
4. Navigate **Plugins** section on the left menu bar.
5. Click **Add new**.

> 6. On the top of the page a **Upload Plugin** button will be shown. Please click it.

Reviewer: ```suggestion
6. On the top of the page a **Upload Plugin** button will be shown. Click it.
```

To make and order with Urb-it just follow the straight steps:

1. Go to the storefront.
2. Place some products to cart.
3. Go to the cart page.

### Cart

1. Choose a shipping option suitable for you (if store is closed at the time, urb-it now option may not appear).
2. Click “Proceed to checkout”.

![Placing an order (cart step)][Img3-Cart]

### Checkout

At the checkout step we need to fill all required billing/shipping information and choose a proper delivery time *(only for urb-it specific time)*. 

1. Fill the “Billing details” form with correct values. The error notification will be shown after pressing “Place order” button If provided values is not correct.
2. **(ONLY FOR URB-IT SPECIFIC TIME)**. Select a delivery day. All available delivery days and time took from the Urb-it API according to X-API-Key (individual for each customer).
3. **(ONLY FOR URB-IT SPECIFIC TIME)**. Select Hour and Minute from the dropdown list when order will be delivered.
4. Push “Place order” button at the bottom of the page.

![Placing an order (checkout step)][Img4-Checkout]

### Checking just received order

> To see the details of the order that we just made, go back to the Wordpress admin panel.. 

Reviewer: ```suggestion
To see the details of the order that we just made, go back to the Wordpress admin panel.
```

> 1. In the left menu bar hover “WooCommerce” section and select “Orders”. 

Reviewer: ```suggestion
1. In the left menu bar, hover “WooCommerce” section and select “Orders”. 
```

2. Choose your order from the list (can be detected by the order number displayed at the “Order recieved” page).

Now you see the order details page. The order type and products are displayed in the “Item” section below, and the delivery time is on the right “Order notes” section. 

There you have two options:

> 1. Wait for the order will be sent to Urb-it by the time picked at checkout (scheduled by the cron task).

Reviewer: ```suggestion
1. Wait for the order to be sent to Urb-it by the time selected at checkout (scheduled by the cron task).
```

> 2. In case when order prepared earlier you can mark it as ready to ship by changing the order status to one specified in Urb-it configuration. Then order will be triggered (cron task will be dropped) and order will go to Urb-it (it will notify Urb-it that it is ready to be shipped).

Reviewer: ```suggestion
2. In case the order has been prepared earlier you can mark it as ready to ship by changing the order status to one specified in Urb-it configuration. Then order will be triggered (cron task will be dropped) and go to Urb-it (it will notify Urb-it that it is ready to be shipped).
```

> To trigger the order please follow the next steps:

Reviewer: ```suggestion
To trigger the order, follow these steps:
```

1. Switch the order status to the “trigger” (see above) status.

> 2. Update order by clicking “Update” button on the right side.

Reviewer: ```suggestion
2. Update the order by clicking the “Update” button on the right side.
```

![Checking and sending an order][Img5-Order]

## Settings reference

> - **Undeliverable Product** : This is the switch to enable a bloc to warn the consumer when a product can’t be delivered via Urb-it

Reviewer: ```suggestion
- **Undeliverable Product** : This is the switch to enable a bloc to warn the customer that a product can’t be delivered via Urb-it.

> - **Exceeded Cart Limit** : This is the switch to enable a bloc to warn the consumer when the cart exceeds the service limits (weight or volume)

Reviewer: ```suggestion
- **Exceeded Cart Limit** : This is the switch to enable a bloc to warn the customer when the cart exceeds the service limits defined by Urb-it (weight or volume).
```

## Troubleshooting

> **If shipping option is not visible at all in the checkout**: check the plugin is installed and activated.

Reviewer: ```suggestion
**If the shipping option is not visible at all in the checkout**: check that the plugin is installed and activated.
```
